src/browser/components/DAppContainer/index.js

```diff
@@ -11,7 +11,8 @@ const mapDispatchToProps = (dispatch) => bindActionCreators({
   setTabLoaded,
   enqueue,
   dequeue,
-  empty
+  empty,
+  closeTab
```

Comment:
:neckbeard: Can this come after `setTabLoaded` just so that contextually all related actions are the same?